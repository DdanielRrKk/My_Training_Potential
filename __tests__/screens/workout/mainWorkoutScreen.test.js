import React from 'react';
import renderer from 'react-test-renderer';

import MainWorkoutScreen from '../../../screens/workout/mainWorkoutScreen';



describe('<MainWorkoutScreen />', () => {
  jest.useFakeTimers();

  it('has 1 child', async () => {
    const tree = await renderer.create(<MainWorkoutScreen />).toJSON();
    expect(tree.children.length).toBe(1);
  });

  it('renders correctly', async () => {
    const tree = await renderer.create(<MainWorkoutScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // it('renders correctly', async () => {
  //   let tree;
    
  //   await renderer.act(async () => { tree = renderer.create(<MainWorkoutScreen />); });

  //   expect(tree.toJSON()).toMatchSnapshot();
  // });
});
