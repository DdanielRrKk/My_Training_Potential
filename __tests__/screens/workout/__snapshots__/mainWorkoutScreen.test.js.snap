// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainWorkoutScreen /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#1b442b",
      }
    }
  >
    Loading
  </Text>
</View>
`;
